### PR TITLE
Fix regex pattern for parsing color

### PR DESCRIPTION
**Pull request by cocopon**
#601

### CODE REVIEW

File: packages/core/src/input-binding/color/converter/color-string.ts [hunk 1]
```diff
@@ -208,7 +208,7 @@ function parseHexRgbColor(text: string): IntColor | null {
 
 function parseHexRgbaColorComponents(text: string): ColorComponents4 | null {
 	const mRgb = text.match(
-		/^#?([0-9A-Fa-f])([0-9A-Fa-f])([0-9A-Fa-f])([0-9A-Fa-f])$/,
+		/^#([0-9A-Fa-f])([0-9A-Fa-f])([0-9A-Fa-f])([0-9A-Fa-f])$/,
```
Comment (Sija) [inline packages/core/src/input-binding/color/converter/color-string.ts:211]: I'd suggest using sth like: `/#([a-f0-9]{6}|[a-f0-9]{3})/`

Comment (cocopon) [reply packages/core/src/input-binding/color/converter/color-string.ts:211]: Have you checked the whole code of this function?
https://github.com/cocopon/tweakpane/blob/80829c4560d19076eb22bf9a513d82286267fcaf/packages/core/src/input-binding/color/converter/color-string.ts#L209-L235

If you feel this is incorrect, you can suggest fixed code and additional test cases via a pull request or sharing the code.

Comment (Sija) [reply packages/core/src/input-binding/color/converter/color-string.ts:211]: I haven't checked the whole function tbh, it seemed to me to be a hex color matching pattern and I offered what I thought is correct version without digging more into the code...

Comment (cocopon) [reply packages/core/src/input-binding/color/converter/color-string.ts:211]: This pattern matching is covered by various test cases, as seen here:
https://github.com/cocopon/tweakpane/blob/80829c4560d19076eb22bf9a513d82286267fcaf/packages/core/src/input-binding/color/converter/color-string-test.ts#L26-L158

If someone says it "incorrect", there must be a failing test case, and this is exactly what I'd like to know as a maintainer.

**Pull request merged**